server/data/tokenStore.ts

```diff
@@ -28,11 +29,43 @@ export default class TokenStore {
     this.setRedisAsync = promisify(redisClient.set).bind(redisClient)
   }
 
-  public async setToken(key: string, token: string, durationSeconds: number): Promise<void> {
-    this.setRedisAsync(key, token, 'EX', durationSeconds)
+  public getAuthToken = async (username?: string): Promise<string> => {
+    const key = username || '%ANONYMOUS%'
+    const token = await this.getToken(key)
+    if (token) {
+      return token
+    }
+
+    const clientToken = generateOauthClientToken(
+      config.apis.hmppsAuth.systemClientId,
+      config.apis.hmppsAuth.systemClientSecret
+    )
```

Comment:
This is always getting a system token.

Reply:
Yes this method will now always get system tokens. The request will not come in here if a user's token is supplied